src/main/java/vectorwing/farmersdelight/common/crafting/ingredient/ChanceResult.java

```diff
@@ -77,7 +77,7 @@ public static ChanceResult deserialize(JsonElement je) {
 		float chance = GsonHelper.getAsFloat(json, "chance", 1);
 		ItemStack itemstack = new ItemStack(ForgeRegistries.ITEMS.getValue(new ResourceLocation(itemId)), count);
 
-		if (GsonHelper.isValidPrimitive(json, "nbt")) {
+		if (json.has("nbt")) {
```

Comment:
Best I can tell, an nbt is not a JsonPrimitive and so Gson.isValidPrimitive never evaluates to true even if the json is properly formatted. I changed it to just check if the nbt is present which I think is sufficient and in line with how forge's Crafting Helper does it

Reply:
Yeah... not sure why it's calling this method. While much of ChanceResult can from Create, they never used this call, apparently.

I might change it to `GsonHelper.isValidNode()` instead, but for now, this one should suffice.